gapic/schema/wrappers.py

```diff
@@ -158,6 +164,44 @@ def inner_mock(self, stack, visited_fields):
         # Done; return the mock value.
         return answer
 
+    def inner_mock_as_original_type(self) -> Union[bool, str, bytes, int, float, List[Any], None]:
```

Comment:
I think it might be simpler and cleaner to rework `mock` and `inner_mock` to return the type and then just convert to string when need be, or make the `_as_original_type` variants the 'real' one and have the others just be thin wrappers. I'm concerned that duplicating much of the logic is going to cause maintenance problems. If this turns out to be non trivial then just ignore this comment.

Reply:
I tried to de-dupe some of th logic in https://github.com/googleapis/gapic-generator-python/pull/941/commits/b5c8e2495ca70112e4aa57c93ee1058f50271e4b. Let me know what you think. 

The complicating factor seems to be that different primitive types need different transformations (strings need to be wrapped with another pair of quotes, ints and floats can just be turned into strings, etc.)  to render correctly in the client library unit test templates.